wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -175,6 +175,39 @@ def test_validate_non_required_richtext_block(self):
         self.assertIsInstance(result, RichText)
         self.assertEqual(result.source, '')
 
+    def test_widget_field_richtext_block(self):
+        from wagtail.wagtailcore.fields import RichTextArea
+        custom_widget = RichTextArea(editor_config={'plugins': {'halloheadings': {'formatBlocks': ['p', 'h2']}}})
+        block = blocks.RichTextBlock(widget=custom_widget)
+        self.assertEqual(block.field_options.get('widget'), custom_widget)
```

Comment:
We don't need to care about internal implementation. So we can just remove this test, I think.